source/bits/lshiftl.ts

```diff
@@ -0,0 +1,21 @@
+import { Bit, Bits } from '../types'
+
+/**
+ * Logical Shift Left
+ * 
+ * Shifts all given bits to left and returns the resulting bits
+ *
+ * @example
+ * lshiftl([1,0,1,1,0,1]) => [0,1,1,0,1,0]
```

Comment:
@0xflotus can you make a case for this not returning `[1,0,1,1,0,1,0]` instead? `bitwise` basically supports arbitrary size bit arrays, so perhaps this is an unexpected result

If you have a particular use-case in mind for the non-size-increasing version, we could also add a flag for this behavior.

Reply:
There are 6 Bits in the example and you are referring to 7 Bits. Which case should it return?